src/materials/CrazyKCPlantFitsBoundary.C

```diff
@@ -16,6 +16,8 @@ InputParameters
 CrazyKCPlantFitsBoundary::validParams()
 {
   InputParameters params = ADMaterial::validParams();
+  params.addClassDescription(
+      "Boundary material properties corresponding to KC plant reference material.");
```

Comment:
Same comment here about making it more generic

Reply:
Changed in 6f7c698